scipy/special/_basic.py

```diff
@@ -685,6 +703,16 @@ def yvp(v, z, n=1):
     .. [2] NIST Digital Library of Mathematical Functions.
            https://dlmf.nist.gov/10.6.E7
 
+    Examples
+    --------
+    Compute the first derivative of the v=2 Bessel function of the second kind  at z=1.
+
+    >>> from scipy import special
+    >>> v=2
+    >>> z=1
+    >>> result = special.yvp(v,z)
```

Comment:
You don't need the intermediate `result` variables. You can directly print the result just as you did with `assoc_laguerre`